src/Definition/RuntimeDefinition.php

```diff
@@ -19,17 +19,15 @@ class RuntimeDefinition implements DefinitionInterface
     /** @var ClassDefinition[] */
     private array $definition = [];
 
-    /** @var bool[] */
-    private ?array $explicitClasses = null;
+    /** @var array<class-string, bool> */
+    private array $explicitClasses;
 
     /**
-     * @param null|string[] $explicitClasses
+     * @param class-string[] $explicitClasses
      */
-    public function __construct(?array $explicitClasses = null)
+    public function __construct(array $explicitClasses = [])
```

Comment:
Ah, the `public` constructor we cannot change: that makes it a BC break :D 

Before, you could give it `null`, and it would internally populate an `array`

Reply:
Current changes revert it back to the original signature with the docblock updated.